test/Passes/pass-tile-linalg-matmul.mlir

```diff
@@ -0,0 +1,50 @@
+
+// Test 1
+// gemm f32
+// RUN: tpp-opt %s --tile-linalg="mTile=4,8 nTile=8,16" | FileCheck %s
```

Comment:
In lit tests, RUN executes for the whole file anyway.
But to separate checks you can add `-split-input-file` flag and separate test cases (functions/modules) with line `// -----`

Reply:
Thanks. Added the `-split-input-file` option.